SOP/SOP-push.adoc

```diff
@@ -78,43 +78,36 @@ unifiedpush-operator-58c8877fd8-g6dvr          1/1       Running   3          9d
 [source,shell]
 ----
  $ oc describe pods example-unifiedpushserver-postgresql-1-bw8mt
-Name:               example-unifiedpushserver-postgresql-1-bw8mt
+Name:               example-unifiedpushserver-postgresql-5d44cbb4f5-sxssg
 Namespace:          unifiedpush
 Priority:           0
 PriorityClassName:  <none>
-Node:               localhost/192.168.64.27
-Start Time:         Wed, 03 Jul 2019 21:45:02 -0300
+Node:               localhost/192.168.122.127
+Start Time:         Thu, 26 Sep 2019 11:48:32 +0300
 Labels:             app=example-unifiedpushserver
-                    deployment=example-unifiedpushserver-postgresql-1
-                    deploymentconfig=example-unifiedpushserver-postgresql
+                    pod-template-hash=1800766091
                     service=example-unifiedpushserver-postgresql
-Annotations:        openshift.io/deployment-config.latest-version=1
-                    openshift.io/deployment-config.name=example-unifiedpushserver-postgresql
-                    openshift.io/deployment.name=example-unifiedpushserver-postgresql-1
-                    openshift.io/scc=restricted
+Annotations:        openshift.io/scc: restricted
 Status:             Running
-IP:                 172.17.0.13
-Controlled By:      ReplicationController/example-unifiedpushserver-postgresql-1
+IP:                 172.17.0.9
+Controlled By:      ReplicaSet/example-unifiedpushserver-postgresql-5d44cbb4f5
 Containers:
   postgresql:
-    Container ID:   docker://16f6f7ea2ac25f72d308d4e89662dc5f4ccd1935db51374aedaed64edd104934
-    Image:          172.30.1.1:5000/openshift/postgresql@sha256:0c78f036478b50800913056c564147ec452214fd0b6d41f4eec4fb3b5c63d246
-    Image ID:       docker-pullable://172.30.1.1:5000/openshift/postgresql@sha256:0c78f036478b50800913056c564147ec452214fd0b6d41f4eec4fb3b5c63d246
+    Container ID:   docker://25372b0d08518630d16e9f9a9a49b051ed1eda57aa8b6602908396a43ca66b78
+    Image:          docker.io/centos/postgresql-96-centos7:9.6
```

Comment:
The value I had here was:  `centos/postgresql-10-centos7:1`

Reply:
oh, right. forgot to update this after a rebase.
let me do this anyway although we will remove the Postgres provisioning within the operator soon

Reply:
We'll still be keeping PostgreSQL provisioning for the time being, just providing an alternative to not use it. While there's still an integreatly on OpenShift v3.11, we'll have it.

Reply:
Oops, yes!